sormas-base/pom.xml

```diff
@@ -215,14 +215,6 @@
 				<scope>provided</scope>
 			</dependency>
 
-			<dependency>
```

Comment:
Either the ticket number is wrong or the branch ...

Reply:
The ticket number is wrong. Will squash the commits when merging and hopefully that will fix the issue

Reply:
No, please override the commit to change the ticket id, but don't squash the history (is better understandable in retrospect imho).

Reply:
Done. Should be ok now